server/src/immich/app.service.ts

```diff
@@ -61,7 +61,12 @@ export class AppService {
   }
 
   ssr(excludePaths: string[]) {
-    const index = readFileSync('/usr/src/app/www/index.html').toString();
+    let index = '';
+    try {
+      index = readFileSync('/usr/src/app/www/index.html').toString();
```

Comment:
Can we use an env or something for this ? It breaks the AIO image

Reply:
I'm fine with adding an `IMMICH_WEB_ROOT` that is set to /usr/src/app/www`

Reply:
Should it be docs ?